pkg/util/label.go

```diff
@@ -13,6 +13,20 @@ func GetLabelValue(labels map[string]string, labelKey string) string {
 	return labels[labelKey]
 }
 
+// MergeLabels merges the labels from 'src' to 'dst'.
+func MergeLabels(dst *unstructured.Unstructured, src *unstructured.Unstructured) {
+	for key, value := range src.GetLabels() {
+		labels := dst.GetLabels()
+		if labels == nil {
+			labels = make(map[string]string, 1)
+		}
+		if _, exist := labels[key]; !exist {
+			labels[key] = value
+			dst.SetLabels(labels)
+		}
+	}
```

Comment:
How about using the `MergeLabel` method directly?

Reply:
The logic is not the same as `MergeLabel`.

Reply:
Oh... Different processing of the same key